### ISSUE TITLE
vtkOrientationMarkerWidget fails to render properly or update

### ISSUE THREAD

**Issue by spookylukey** [Kitware/trame#412]
**Describe the bug**

Using  [vtkOrientationMarkerWidget](https://examples.vtk.org/site/Python/Visualization/DisplayCoordinateAxes/) with trame, the orientation widget does not display properly or update.

I think [vtkCameraOrientationWidget](https://examples.vtk.org/site/Python/Widgets/CameraOrientationWidget/) is similarly broken (it fails to display at all) but I haven't investigated fully.

**To Reproduce**

See the code below, which produces the following screenshot:

![Screenshot from 2024-01-12 15-46-12](https://github.com/Kitware/trame/assets/62745/789c98c0-cf98-424a-b763-eb2c5e18bb16)

If I uncomment out the lines indicated (this doesn't make sense, just for debugging this), then I first get an interactive window, with the axes displaying properly. After closing this, the trame app appears in a browser window, looking like this:

![Screenshot from 2024-01-12 15-55-30](https://github.com/Kitware/trame/assets/62745/ab39a191-0b79-4584-8eda-c6f513bcfe4b)

So the axes look correct here, but they still don't move when you change the camera position with the mouse.

***Code***
The code here is based on a minimal trame app (from https://github.com/Kitware/trame-tutorial/blob/master/01_vtk/solution_cone.py ) combined with  https://examples.vtk.org/site/Python/Visualization/DisplayCoordinateAxes/

```python
from trame.app import get_server
from trame.ui.vuetify import SinglePageLayout
from trame.widgets import vtk, vuetify

from vtkmodules.vtkFiltersSources import vtkConeSource
from vtkmodules.vtkRenderingCore import (
    vtkActor,
    vtkPolyDataMapper,
    vtkRenderer,
    vtkRenderWindow,
    vtkRenderWindowInteractor,
)

# Required for interactor initialization
from vtkmodules.vtkInteractionStyle import vtkInteractorStyleSwitch  # noqa

# Required for rendering initialization, not necessary for
# local rendering, but doesn't hurt to include it
import vtkmodules.vtkRenderingOpenGL2  # noqa

from vtkmodules.vtkInteractionWidgets import vtkOrientationMarkerWidget
from vtkmodules.vtkRenderingAnnotation import vtkAxesActor

renderer = vtkRenderer()
renderWindow = vtkRenderWindow()
renderWindow.AddRenderer(renderer)

renderWindowInteractor = vtkRenderWindowInteractor()
renderWindowInteractor.SetRenderWindow(renderWindow)
renderWindowInteractor.GetInteractorStyle().SetCurrentStyleToTrackballCamera()

cone_source = vtkConeSource()
mapper = vtkPolyDataMapper()
mapper.SetInputConnection(cone_source.GetOutputPort())
actor = vtkActor()
actor.SetMapper(mapper)

renderer.AddActor(actor)

axes = vtkAxesActor()

widget = vtkOrientationMarkerWidget()
widget.SetOrientationMarker(axes)
widget.SetInteractor(renderWindowInteractor)
widget.SetViewport(0.0, 0.0, 0.4, 0.4)
widget.SetEnabled(1)
widget.InteractiveOn()

renderer.GetActiveCamera().Azimuth(50)
renderer.GetActiveCamera().Elevation(-30)

renderer.ResetCamera()

# Un-comment these to show a normal VTK app with desired behavior:
# renderWindow.Render()
# renderWindowInteractor.Start()


server = get_server(client_type="vue2")
ctrl = server.controller

with SinglePageLayout(server) as layout:
    layout.title.set_text("Hello trame")

    with layout.content:
        with vuetify.VContainer(
            fluid=True,
            classes="pa-0 fill-height",
        ):
            view = vtk.VtkLocalView(renderWindow)

if __name__ == "__main__":
    server.start()

```

**Expected behavior**

It should display an axes that moves when you move the camera around.

**Screenshots**

If applicable, add screenshots to help explain your problem (drag and drop the image).

**Platform:**

I have ticked only those that I have checked, but based on Chrome+Firefox on both Windows+Linux being affected and the symptoms, I expect all platforms/devices are affected.

***Device:***

- [X] Desktop
- [ ] Mobile

***OS:***

- [x] Windows
- [ ] MacOS
- [X] Linux
- [ ] Android
- [ ] iOS

***Browsers Affected:***

- [X] Chrome
- [X] Firefox
- [x] Microsoft Edge
- [ ] Safari
- [ ] Opera
- [ ] Brave
- [ ] IE 11

**Comment by jourdain** [comment]
You need to provide the [`widgets=[...]`](https://github.com/Kitware/trame-vtk/blob/master/trame_vtk/widgets/vtk/common.py#L813) argument.

**Comment by spookylukey** [comment]
@jourdain Thanks for the pointer!

To be clear for the next person, the following code got it to work:

```python
view = vtk.VtkLocalView(renderWindow, widgets=[widget])
```
where `widget` is the `vtkOrientationMarkerWidget` instance.

As far as I can see, this isn't documented anywhere  e.g. https://trame.readthedocs.io/en/latest/trame.widgets.vtk.html#trame.widgets.vtk.VtkLocalView  .

**Comment by jourdain** [comment]
True, it is listed on the API but not explained. Also, not all the widgets are supported. 
It is an advanced feature that still needs to mature to be really considered mainstream.
